### PR TITLE
Laboratory UI changes

### PR DESCRIPTION
**Pull request by FuriousLlama**
## Status
**Ready**

## Fixes or Implements [AB#8374](https://qslvic.visualstudio.com/304a1f8c-dace-4f85-adf3-bf563d5b3a39/_workitems/edit/8374) & [AB#8381](https://qslvic.visualstudio.com/304a1f8c-dace-4f85-adf3-bf563d5b3a39/_workitems/edit/8381)
- [x] Enhancement
- [ ] Bug
- [ ] Documentation

## Description:
Changed the out-of-range to be Yes/No
Report summary not only shows when available
Report summary is visible on the summary view

## Testing
- [ ] Unit Tests Updated
- [ ] Functional Tests Updated
- [X] Not Required

### UI Changes
YES

### Browsers Tested
- [X] Chrome
- [ ] Safari
- [ ] Edge
- [ ] Firefox

### CODE REVIEW

File: Apps/WebClient/src/ClientApp/app/components/timeline/laboratory.vue [hunk 1]
```diff
@@ -242,5 +247,9 @@ export default class LaboratoryTimelineComponent extends Vue {
         this.isLoadingDocument = false;
       });
   }
+
+  private get reportAvailiable(): boolean {
```
Comment (FuriousLlama) [inline Apps/WebClient/src/ClientApp/app/components/timeline/laboratory.vue:251]: Redundant for flexibility. Will be removed once we get confirmation that the flag is pass uniquely.

File: Apps/Laboratory/src/Models/LaboratoryOrder.cs [hunk 2]
```diff
@@ -84,6 +84,12 @@ public class LaboratoryOrder
         [JsonPropertyName("additionalData")]
         public string AdditionalData { get; set; } = string.Empty;
 
+        /// <summary>
+        /// Gets or sets additional related data.
+        /// </summary>
+        [JsonPropertyName("reportAvailable")]
```
Comment (sslaws) [inline Apps/Laboratory/src/Models/LaboratoryOrder.cs:90]: Is this being added to the PHSA model?

File: Apps/WebClient/src/ClientApp/app/models/laboratoryTimelineEntry.ts [hunk 3]
```diff
@@ -80,7 +82,7 @@ export class LaboratoryResultViewModel {
   constructor(model: LaboratoryResult) {
     this.id = model.id;
     this.testType = model.testType;
-    this.outOfRange = model.outOfRange;
+    this.outOfRange = model.outOfRange ? "True" : "False";
```
Comment (sslaws) [inline Apps/WebClient/src/ClientApp/app/models/laboratoryTimelineEntry.ts:85]: This should be a a view level change and not a model change.

File: Apps/WebClient/src/ClientApp/app/views/landing.vue [hunk 4]
```diff
@@ -468,7 +468,7 @@ import {
   faSyringe,
   IconDefinition,
   faClipboard,
-  faExclamationTriangle,
+  faExclamationTriangle
```
Comment (sslaws) [inline Apps/WebClient/src/ClientApp/app/views/landing.vue:471]: Were the commas removed from linting?

File: Apps/Laboratory/src/Models/LaboratoryOrder.cs (hunk 2 above)
Comment (FuriousLlama) [reply Apps/Laboratory/src/Models/LaboratoryOrder.cs:90]: AFAIK yes

File: Apps/WebClient/src/ClientApp/app/models/laboratoryTimelineEntry.ts (hunk 3 above)
Comment (FuriousLlama) [reply Apps/WebClient/src/ClientApp/app/models/laboratoryTimelineEntry.ts:85]: thisis the view model

File: Apps/WebClient/src/ClientApp/app/views/landing.vue (hunk 4 above)
Comment (FuriousLlama) [reply Apps/WebClient/src/ClientApp/app/views/landing.vue:471]: correct. Not sure whats up with sams linter

File: Apps/WebClient/src/ClientApp/app/models/laboratoryTimelineEntry.ts (hunk 3 above)
Comment (sslaws) [reply Apps/WebClient/src/ClientApp/app/models/laboratoryTimelineEntry.ts:85]: I'll give it to your for the smiley!  Most of the other code has it mapped in the actual view and not in the model.

**Pull request merged**